lib/promenade/client/rack/collector.rb

```diff
@@ -28,11 +28,13 @@ class Collector
         end
         private_constant :DEFAULT_LABEL_BUILDER
 
-        DEFAULT_EXCEPTION_HANDLER = proc do |exception, counter|
+        # rubocop:disable Lint/UnusedBlockArgument
+        DEFAULT_EXCEPTION_HANDLER = proc do |exception, counter, env, duration|
           counter.increment(exception: exception.class.name)
```

Comment:
I think this a behaviour that by default we want to track on`durations_histogram`

Reply:
Sorry I think what we want is to track the number of request per HTTP code and path so we can query `http_exceptions_total{code="500", path="/posts"}`, likewise we want to track the duration of all requests `http_req_duration_seconds` otherwise you might not notice high latency caused by many slow but failing requests